### PR TITLE
Add github teacher's San Diego file

### LINKED ISSUE #21: githubteacher's Workflow for adding San Diego to the Repository
## San Diego, CA
- [x] Create an Issue
- [x] Create a branch
- [x] Make changes
- [x] Open a PR
- [x] Collaborate/Have a discussion
- [x] Get approval to 🚢 ( 👍 ) 
- [x] Merge

### PR DESCRIPTION
**Pull request by githubteacher**
closes #21

### CODE REVIEW

File: githubteacher-sandiego.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+
+### Restaurants
+- City Tacos
+- Poke One n Half
+
+### Places to Go
+- Balboa Park
+
+### Current Temperature
+- 94 degrees
```
Comment (githubteacher) [inline githubteacher-sandiego.md:12]: That's 🔥

File: githubteacher-sandiego.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
+## San Diego, CA
+
+### Restaurants
+- City Tacos
+- Poke One n Half
+
+### Places to Go
+- Balboa Park
```
Comment (githubteacher) [inline githubteacher-sandiego.md:8]: That place is the best.

**Review by githubteacher — COMMENTED** [review]
San Diego sounds so cool!

File: githubteacher-sandiego.md (hunk 1 above)
Comment (brianamarie) [reply githubteacher-sandiego.md:12]: That is horrible...the high today in Chicago is is 67. 😎

**Review by elexpander — APPROVED** [review]

**Pull request merged**